style/style.css

```
@@ -17,26 +17,43 @@
 }
 
 body {
-    min-width: 960px;
+    min-width: 1200px;
     margin: 0;
     padding: 0;
     font-family: "Roboto", "Arial", sans-serif;
     font-size: 16px;
     line-height: 24px;
     color: var(--basic-black);
-    background-color: #ffffff;
+    background-color: var(--basic-white);
 }
 
+.container {
+    width: 1160px;
+    padding-left: 20px;
+    padding-right: 20px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
 .main-header {
     background-color: var(--basic-grey);
     padding-top: 30px;
-    padding-left: 100px;
-    padding-right: 100px;
+}
+
+.logo:hover {
+    opacity: 0.7;
+}
+
+.logo:active {
+    opacity: 0.3;
 }
 
 .main-navigation {
-    display: flex;
-    justify-content: space-between;
+    display: grid;
+    grid-template-columns: 160px min-content 115px;
```

В навигации лучше использовать такой шаблон:
```suggestion
    grid-template-columns: 160px min-content 160px;
```
так как у пункта корзины есть иконка.